### PR TITLE
Add hello singularity

### LINKED ISSUE #18: Hello World singularity image
Include a Hello World singularity image to explain basic steps

### PR DESCRIPTION
**Pull request by amanmdesai**
Closes #18

### CODE REVIEW

File: _episodes/04-building-containers.md
Comment (klieret) [inline _episodes/04-building-containers.md]: I think we've started using 

    ```bash

now (just to keep it standard). If you use this markup, it'd be `.language-bash` instead of `.source` but it's probably to keep everything standardized anyway

File: _episodes/04-building-containers.md
Comment (klieret) [inline _episodes/04-building-containers.md]: You can try also simply doing this with 

    ```source

but I'm not sure if this works. Else just leave this as-is :)

File: _episodes/04-building-containers.md
Comment (klieret) [inline _episodes/04-building-containers.md]: This and the next should be with 

    ```bash

again

File: _episodes/04-building-containers.md
Comment (klieret) [inline _episodes/04-building-containers.md]: Should we use `nano` here? In case people run this on a server?

Maybe also add "a text editor (e.g., `nano`, `vim`, or `gedit` in a graphical environment)"

**Review by klieret — CHANGES_REQUESTED** [review]

File: _episodes/04-building-containers.md
Comment (amanmdesai) [reply _episodes/04-building-containers.md]: I think this needs to standardized for this repository.

Comment (amanmdesai) [reply _episodes/04-building-containers.md]: applied your suggestions for the hello docker part

File: _episodes/04-building-containers.md
Comment (amanmdesai) [reply _episodes/04-building-containers.md]: using ```bash

File: _episodes/04-building-containers.md
Comment (amanmdesai) [reply _episodes/04-building-containers.md]: done

File: _episodes/04-building-containers.md
Comment (amanmdesai) [reply _episodes/04-building-containers.md]: done

**Comment by amanmdesai** [comment]
@klieret I have updated the bash blocks

File: _episodes/04-building-containers.md
Comment (klieret) [inline _episodes/04-building-containers.md]: Does that format OK? (because it's not actually `bash` content...)

**Review by klieret — APPROVED** [review]

**Comment by klieret** [comment]
Let's wait for @michmx review before we merge though

File: _episodes/04-building-containers.md
Comment (michmx) [inline _episodes/04-building-containers.md]: Thanks a lot, @amanmdesai ! However, my impression here is this "Hello World" container fits better in the next chapter, when containers are built from definition files.

Please include this example in chapter 5 instead of here, where we focus on interactive sessions.

**Review by michmx — CHANGES_REQUESTED** [review]

**Comment by amanmdesai** [comment]
done. 

i realised that my commit message still use "docker" instead of singularity. sorry about that.

File: _episodes/04-building-containers.md
Comment (amanmdesai) [reply _episodes/04-building-containers.md]: Yeah locally `...` works.

**Comment by amanmdesai** [comment]
@michmx this PR ready for re-review

File: _episodes/04-building-containers.md
Comment (amanmdesai) [reply _episodes/04-building-containers.md]: Thanks @michmx. I have included this example in chapter 5 now.

File: _episodes/05-definition-files.md
Comment (michmx) [inline _episodes/05-definition-files.md:54]: Can we add a title here to separate the "hello world" from the second example?  Something like
```
## A more elaborated definition file

**Review by michmx — CHANGES_REQUESTED** [review]

File: _episodes/05-definition-files.md
Comment (amanmdesai) [reply _episodes/05-definition-files.md:54]: Done

**Comment by amanmdesai** [comment]
@michmx  fixed as per your suggestion

**Review by michmx — APPROVED** [review]

**Pull request merged**
